### PR TITLE
Fix for #5855

### PR DESCRIPTION
**Pull request by tstauder**
Changed batch_time to batch_id in test_batch_path_differ function in test_feedexport: #5855 

Fixes https://github.com/scrapy/scrapy/issues/5855

### CODE REVIEW

**Comment by Gallaecio** [comment]
> https://github.com/scrapy/scrapy/issues/5855#issuecomment-1509289786

@tstauder First suggestion: review your changes as [seen in your pull request](https://github.com/scrapy/scrapy/pull/5898/files), and make sure they only include intended changes. I see some commented out code that you probably used during debugging and should not be a part of this change. I also see a bad edit of a docstring.

**Comment by tstauder** [comment]
Thanks for the helpful feedback. I cleaned up those issues. I've update the pull request to include them.

File: tests/test_feedexport.py [hunk 1]
```diff
@@ -2398,7 +2398,7 @@ def test_export_items(self):
         yield self.assertExported(items, header, rows, settings=settings)
 
     def test_wrong_path(self):
-        """If path is without %(batch_time)s and %(batch_id) an exception must be raised"""
+        """If path is without %(batch_id) an exception must be raised"""
```
Comment (Gallaecio) [inline tests/test_feedexport.py:2401]: This change does not seem necessary.

**Review by Gallaecio — APPROVED** [review]
Other than the minor issue I mention below, it looks good to me. Thanks!

File: tests/test_feedexport.py (hunk 1 above)
Comment (felipeboffnunes) [reply tests/test_feedexport.py:2401]: Click on accept suggestion for a free merge!

```suggestion
        """If path is without %(batch_time)s and %(batch_id) an exception must be raised"""
```

Comment (tstauder) [reply tests/test_feedexport.py]: Just saw this suggestion now, but I also added commit that does the same thing

**Review by felipeboffnunes — APPROVED** [review]

**Pull request merged**